docs/en/api/POSIX-API.md

```diff
@@ -73,181 +73,363 @@ For example, running the following commands from the `${ALLUXIO_HOME}` directory
 Alluxio path `/people` to the folder `/mnt/people` on the local file system.
 
 ```console
+# Create the Alluxio folder to be mounted
 $ ${ALLUXIO_HOME}/bin/alluxio fs mkdir /people
+
+# Prepare the local folder to mount Alluxio to
```

Comment:
This sentence doesn't sound natural. Something like "Prepare the local directory to be mounted by Alluxio" might be better?